test/e2e/common/node/pod_resize.go

```diff
@@ -1006,7 +1006,7 @@ func doPodResizeTests(f *framework.Framework) {
 			}
 
 			ginkgo.By("deleting pod")
-			framework.ExpectNoError(podClient.Delete(ctx, newPod.Name, metav1.DeleteOptions{}))
+			podClient.DeleteSync(ctx, newPod.Name, metav1.DeleteOptions{}, f.Timeouts.PodDelete)
```

Comment:
we are ok to skip checking the error here?

Reply:
We dont ignore the error , DeleteSync handles it internally. 
https://github.com/kubernetes/kubernetes/blob/c9092f69fc0c099062dd23cd6ee226bcd52ec790/test/e2e/framework/pod/pod_client.go#L217-L225

Reply:
Nice thanks @esotsal